Bottom: 34b35f0361ac84a57faba0a8e623fb62826734af
Top:    431822c371b7eaaac0def0a2cd154d3fc6ee93bb
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-16 13:18:10 -0500

Refresh of add-buildbot-ntel-worker

---
diff --git a/roles/buildbot-worker-ntel/defaults/main.yml b/roles/buildbot-worker-ntel/defaults/main.yml
index afd6f6b3..f1bd9b16 100644
--- a/roles/buildbot-worker-ntel/defaults/main.yml
+++ b/roles/buildbot-worker-ntel/defaults/main.yml
@@ -23,7 +23,7 @@ config_hostdir: '{{ docker_restore_config_base_dir }}/{{ container_name }}'
 cache_path: '/cache'
 
 # name of the cache volume
-cache_volume: '{{ docker_name_prefix }}{{ hostname }}-cache'
+cache_volume: '{{ docker_name_prefix }}{{ container_hostname }}-cache'
 
 # environment passed to the container
 container_env:
@@ -33,8 +33,11 @@ container_env:
   BUILDBOT_WORKER_NAME: 'worker-ntel'
   BUILDBOT_WORKER_PASS: 'pass'
 
+# container_hostname
+container_hostname: '{{ role_name }}'
+
 # the name of the container being started
-container_name: '{{ docker_name_prefix }}{{ hostname }}'
+container_name: '{{ docker_name_prefix }}{{ container_hostname }}'
 
 # network configuration for the confainer
 container_networks: []
@@ -52,10 +55,7 @@ container_volumes:
 data_path: '/buildbot'
 
 # name of the data volume
-data_volume: '{{ docker_name_prefix }}{{ hostname }}-data'
-
-# hostname
-hostname: '{{ role_name }}'
+data_volume: '{{ docker_name_prefix }}{{ container_hostname }}-data'
 
 # address of hsm cryptographic service
 hsm_host: '{{ansible_facts.default_ipv4.address}}'
@@ -68,10 +68,10 @@ image_args:
   PTK_HSM_HOST: '{{hsm_host}}'
 
 # directory of the image source
-image_dir: '{{ docker_projects_dir }}/{{ hostname }}'
+image_dir: '{{ docker_projects_dir }}/{{ container_hostname }}'
 
 # name of the image being built
-image_name: '{{ docker_registry_username }}/{{ hostname }}'
+image_name: '{{ docker_registry_username }}/{{ container_hostname }}'
 
 # repository URI of the image source
 image_repo: '{{ buildbot_worker_ntel_image_repo | default("https://github.com/novatechweb/docker-buildbot-worker-ntel.git") }}'
diff --git a/roles/buildbot-worker-ntel/tasks/main.yml b/roles/buildbot-worker-ntel/tasks/main.yml
index 86403896..497956b8 100644
--- a/roles/buildbot-worker-ntel/tasks/main.yml
+++ b/roles/buildbot-worker-ntel/tasks/main.yml
@@ -69,8 +69,9 @@
     buildargs: '{{ image_args }}'
     force: true
 
-- name: Start buildbot container
+- name: Start buildbot worker container
   docker_container:
+    hostname: '{{ container_hostname }}'
     name: '{{ container_name }}'
     image: '{{ image_name }}:{{ docker_image_tag }}'
     networks: '{{ container_networks }}'
@@ -86,5 +87,5 @@
 
 - name: Add buildbot_worker_ntel host
   add_host:
-    name: "{{ hostname }}"
+    name: "{{ container_hostname }}"
     pubkey: "{{ output.stdout }}"
